src/App.js

```diff
@@ -1,10 +1,16 @@
 import "./App.css";
+import { LandingPage } from "./pages";
+
+
+
 
 
 function App() {
   return (
     <div className="App">
```

Comment:
try to be consistent with the spaces or formatting. It increases readability and code also looks good